sql/json-functions.md

```diff
@@ -1,26 +1,68 @@
+| [JSON_CONTAINS(target, candidate[, path])][json_contains] | 通过返回 1 或 0 来表示目标 JSON 文档中是否包含给定的 candidate JSON 文档 |
+| [JSON_CONTAINS_PATH(json_doc, one_or_all, path[, path] ...)][json_contains_path] | 通过返回 0 或 1 来表示一个 JSON 文档在给定路径是否包含数据 |
 | [JSON_EXTRACT(json_doc, path[, path] ...)][json_extract]          | 从 JSON 文档中解出某一路径对应的子文档                     |
-| [JSON_UNQUOTE(json_val)][json_unquote]                             | 去掉 JSON 文档外面的引号                                   |
-| [JSON_TYPE(json_val)][json_type]                                   | 检查某 JSON 文档内部内容的类型                             |
-| [JSON_SET(json_doc, path, val[, path, val] ...)][json_set]         | 在 JSON 文档中为某一路径设置子文档                         |
-| [JSON_INSERT(json_doc, path, val[, path, val] ...)][json_insert]   | 在 JSON 文档中在某一路径下插入子文档                       |
-| [JSON_REPLACE(json_doc, path, val[, path, val] ...)][json_replace] | 替换 JSON 文档中的某一路径下的子文档                       |
-| [JSON_REMOVE(json_doc, path[, path] ...)][json_remove]             | 移除 JSON 文档中某一路径下的子文档                         |
-| [JSON_MERGE(json_doc, json_doc[, json_doc] ...)][json_merge]       | 将多个 JSON 文档合并成一个文档，其类型为数组               |
-| [JSON_OBJECT(key, val[, key, val] ...)][json_object]               | 根据一系列 K/V 对创建一个 JSON 文档                        |
-| [JSON_ARRAY([val[, val] ...])][json_array]                         | 根据一系列元素创建一个 JSON 文档                           |
-| ->                                                                 | JSON_EXTRACT(doc, path_literal) 的语法糖                   |
-| ->>                                                                | JSON_UNQUOTE(JSONJSON_EXTRACT(doc, path_literal)) 的语法糖 |
+| [->][json_short_extract]  | 返回执行路径后面的 JSON 列的值；`JSON_EXTRACT(doc, path_literal)` 的语法糖  |
+| [->>][json_short_extract_unquote]  | 返回执行路径后面的 JSON 列的值和转义后的结果； `JSON_UNQUOTE(JSON_EXTRACT(doc, path_literal))` 的语法糖 |
+
+## 修改 JSON 值的函数
+
+| 函数及语法糖 | 功能描述 |
+| --------------------------------- | ----------- |
+| [JSON_INSERT(json_doc, path, val[, path, val] ...)][json_insert] | 在 JSON 文档中在某一路径下插入子文档 |
+| [JSON_MERGE(json_doc, json_doc[, json_doc] ...)][json_merge]  | 将多个 JSON 文档合并成一个文档，其类型为数组 |
+| [JSON_REMOVE(json_doc, path[, path] ...)][json_remove]    | 移除 JSON 文档中某一路径下的子文档 |
+| [JSON_REPLACE(json_doc, path, val[, path, val] ...)][json_replace] | 替换 JSON 文档中的某一路径下的子文档 |
+| [JSON_SET(json_doc, path, val[, path, val] ...)][json_set]  | 在 JSON 文档中为某一路径设置子文档 |
+| [JSON_UNQUOTE(json_val)][json_unquote] |  去掉 JSON 文档外面的引号 |
+
+## 返回 JSON 值属性的函数
+
+| 函数及语法糖 | 功能描述 |
+| --------------------------------- | ----------- |
+| [JSON_LENGTH(json_doc[, path])][json_length] | 返回 JSON 文档的长度；如果路径参数已定，则返回该路径下值的长度 |
+| [JSON_TYPE(json_val)][json_type] | 检查某 JSON 文档内部内容的类型 |
+
+## 未支持的函数
+ 
+TiDB 暂未支持以下 JSON 函数。相关进展参见 [TiDB #7546](https://github.com/pingcap/tidb/issues/7546):
+
+* `JSON_APPEND` 及其别名 `JSON_ARRAY_APPEND`
+* `JSON_ARRAY_INSERT`
+* `JSON_DEPTH`
+* `JSON_KEYS`
```

Comment:
As what Morgan has mentioned, this function has been removed from the unsupported list. See https://github.com/pingcap/docs/pull/680